src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]); // Why does it have to be called selectedTabId? It isn't an id at all
```

Comment:
Because it should actually be an id
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id); // Why does it have to be called selectedTabId? It isn't an id at all
```

Reply:
It cannot be an id, when it is an id the Cypress test fails. It specifically requires it to be Object{3}

Reply:
Where exactly?

Reply:
This should be an id, and based on that id you should find a tab with this title and write a function `onTabSelected` that sets the id of a tab instead of an entire tab

Reply:
The test requires the setting function to use the entire tab as an object, which in turn means the state needs to use the tab, and not an id